src/main/java/org/jabref/model/util/DummyFileUpdateMonitor.java

```diff
@@ -9,12 +8,17 @@
  */
 public class DummyFileUpdateMonitor implements FileUpdateMonitor {
     @Override
-    public void addListenerForFile(Path file, FileUpdateListener listener) throws IOException {
+    public void addListenerForFile(Path file, FileUpdateListener listener) {
```

Comment:
The exception was removed here, but not in the interface. I think, it should also be removed here (and the JavaDoc be adapted).

With the current implementation, it does not throw an exception if the file does not exist.